pkg/relayer/cmd/deps.go

```diff
@@ -40,20 +40,38 @@ func setupRelayerDependencies(
 	queryNodeGRPCUrl := relayMinerConfig.PocketNode.QueryNodeGRPCUrl
 	txNodeRPCUrl := relayMinerConfig.PocketNode.TxNodeRPCUrl
 
+	nodeRPCURL, err := cmd.Flags().GetString(cosmosflags.FlagNode)
+	if err != nil {
+		return nil, err
+	}
+
+	nodeGRPCURL, err := cmd.Flags().GetString(cosmosflags.FlagGRPC)
+	if err != nil {
+		return nil, err
+	}
+
 	// Override config file's `QueryNodeGRPCUrl` with `--grpc-addr` flag if specified.
 	// TODO(#223): Remove this check once viper is used as SoT for overridable config values.
```

Comment:
Can we remove this?

I don't see us "planning" to use viper.

If/when it comes up as a priority, we'll deal with it. 

For now, I think we should just remove the TODO. We can add more details to the comment if need be.

Reply:
I don't have a strong preference; however I believe that viper is already in use by cosmos -sdk.